lib/listing.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
+class Listing
+
+  def self.create(title)
+    if ENV['RACK_ENV'] == 'test'
```

Comment:
Perhaps add a new DBConnection class, where you have the if statement so we don't have this connection here as it looks repetitive in the  `.all` method